RecoTracker/MkFitCore/interface/Hit.h

```diff
@@ -218,19 +222,19 @@ namespace mkfit {
 
     static unsigned int minChargePerCM() { return kMinChargePerCM; }
     static unsigned int maxChargePerCM() { return kMinChargePerCM + (0xfe << 3); }
```

Comment:
here 0xfe is supposedly `(1<<maxChargePerCM)-2`
there is also an `xff` above

Reply:
Done. I am not sure why here it's -2, though.

Reply:
that's the max value that could appear in the `get_charge_pcm` return
`          return ((charge_pcm - 1) << 3) + kMinChargePerCM;`

Reply:
Right: `-1` in `charge_pcm` and `-1` in `get_charge_pcm`, summing to `-2`. Makes sense.